dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -783,9 +783,9 @@
             "modelAsString": true
           }
         },
-        "documentType": {
+        "healthcareDocumentType": {
```

Comment:
The name of the property should probably be kept the same, and the name of the enum is that only thing that changes.